landscape/prod/maps/sites.map

```diff
@@ -560,6 +560,7 @@ _/coi content ;
 _/collegechampion content ;
 _/com-csc content ;
 _/com-dean-search content ;
+_/com/amazeen-review/ content ;
```

Comment:
I'm pretty sure this has to be `_/com/amazeen-review` without the trailing slash